setup.cfg

```diff
@@ -107,28 +107,29 @@ optional =
     triangle
 testing =
     babel>=2.9.0
-    pytest>=7.0.0
-    pytest-qt
-    hypothesis>=6.8.0
-    xarray
     fsspec
-    zarr
+    hypothesis>=6.8.0
+    lxml!=4.9.0
```

Comment:
Unrelated to this PR though maybe a good time to change it.

We recently changed this version constraint to prevent a test from failing with `lxml==4.9.0` (see https://github.com/napari/napari/pull/4616 and https://napari.zulipchat.com/#narrow/stream/212875-general/topic/Parsing.20alt.20text.20tests.20failing.20on.20ubuntu.2FmacOS.20CI)

I opened https://github.com/napari/napari/pull/4617 that might fix the underlying issue and allow us to use `lxml==0.4.9`, though that version at least has some inconsistency with some input we give it. 

https://github.com/napari/napari/pull/4641 was recently merged which I think will also allow us to use `lxml==0.4.9` because you removed the leading `<` character in one of the test inputs.

So, I think we should either remove the `!=` here now or add it the `docs` section to be consistent (i.e. test what we actually use to build the docs).

Reply:
sorry for the overlap with your pr there.  let's just remove the != ... for an "optional" feature there (i.e. we don't depend on lxml, it's up to the user, and they can get whatever version they want), we can't really pin our tests just so that it works/.

I think that test just needed a little improving.  I still think your unescape->escape looks correct